package.json

```diff
@@ -17,6 +17,7 @@
     "@ethersproject/strings": "^5.0.5",
     "@ethersproject/units": "^5.0.3",
     "@lottiefiles/svelte-lottie-player": "^0.2.0",
+    "@microsoft/fetch-event-source": "^2.0.1",
```

Comment:
Do we need a whole package for this?

Reply:
We don't since at the moment zapper accepts the api key inside the url, later this will not be the case, this will easily allow to pass custom headers to sse calls afaik.
I am happy to refactor without library but this gives much flexibility to the SSE api.